<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UComponentLinksWidget</class>
 <widget class="QWidget" name="UComponentLinksWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Outputs</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="labelOutputsComponentName">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeWidgetOutputs">
          <property name="rootIsDecorated">
           <bool>true</bool>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerHighlightSections">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">Type</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Inputs</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="labelInputsComponentName">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeWidgetInputs">
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerHighlightSections">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>is Connected</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Links</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="treeWidgetLinks">
         <column>
          <property name="text">
           <string>Ouput component</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Output name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Input component</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Input name</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBreakLink">
       <property name="text">
        <string>Break Link</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCreateLink">
       <property name="text">
        <string>Create Link</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
